controllers/user.controller.js

```diff
@@ -77,66 +95,104 @@ const getUserById = async (req, res) => {
     });
   } catch (error) {
     console.log(error);
-    res.status(httpStatus.INTERNAL_SERVER_ERROR).json({
+    return res.status(httpStatus.INTERNAL_SERVER_ERROR).json({
```

Comment:
```suggestion
    res.status(httpStatus.INTERNAL_SERVER_ERROR).json({
```
vào catch rồi không còn case nào tiếp theo nên không cần return => bỏ